File: Gym/app/src/main/java/com/example/gym/models/Specialization.kt
package com.example.gym.models

import com.google.gson.annotations.SerializedName

data class Specialization (
        @SerializedName("idSpecialization") val idSpecialization : Int,
        @SerializedName("name") val name : String,
        @SerializedName("image") val image : String
)

File: Gym_.NET-master/Gym.API/Domain/Models/Specialization.cs
using System.Collections.Generic;

namespace Gym.API.Domain.Models
{

    public class Specialization
    {
        public int IdSpecialization { get; set; }

        public string Name { get; set; }
        public string Image { get; set; }

        public IList<ScheduleTraining> ScheduleTraining { get; set; } = new List<ScheduleTraining>();

    }
}